### PR TITLE
Allow AfHER to run as an Arches App

### PR DESCRIPTION
**Pull request by chiatt**
Allows AfHER to run as an Arches App

### CODE REVIEW

File: docker/aher_project/docker/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "build_production": "./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 NODE_ENV=production ./media/node_modules/.bin/webpack --config webpack/webpack.config.prod.js",
+        "build_development": "./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack --config webpack/webpack.config.dev.js",
+        "build_test": "./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack --config webpack/webpack.config.dev.js --env test=true",
+        "start": "./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack serve --config webpack/webpack.config.dev.js"
+    },
+    "devDependencies": {
+        "arches-dev-dependencies": "archesproject/arches-dev-dependencies#stable/7.5.4"
+    },
+    "dependencies": {
+        "arches": "archesproject/arches#stable/7.5.4",
```
Comment (aj-he) [inline docker/aher_project/docker/package.json]: @chiatt - this needs updating to 7.5.5

```
"devDependencies": {
        "arches-dev-dependencies": "archesproject/arches-dev-dependencies#stable/7.5.5" <----
    },
    "dependencies": {
        "arches": "archesproject/arches#stable/7.5.5", <----
        "arches_her": "archesproject/arches_her#dev/1.0.x"
    },
```

File: arches_her/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,572 @@
...
+        ]):
+            search_component.delete()
+
+
+    add_map_source = """
+        INSERT INTO map_sources(name, source)
+            VALUES ('select-application-area', '{
+                "data": "/geojson?nodeid=1909956f-3a3b-11eb-ae99-f875a44e0e11&include_geojson_link=true",
+                "type": "geojson"
+            }');
```
Comment (aj-he) [inline arches_her/migrations/0001_initial.py]: This was recently changed for the new application area MVT endpoint

```suggestion
INSERT INTO map_sources(name, source)
    VALUES ('select-application-area', '{
        "data": "/geojson?nodeid=1909956f-3a3b-11eb-ae99-f875a44e0e11", 
        "type": "vector", 
        "tiles": ["/application-areas/{z}/{x}/{y}.pbf"], 
        "minzoom": 6
    }');
```

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+[project]
+name = "arches-her"
+readme = "README.md"
+authors = [
+    {name = "Getty Conservation Institute", email = "contact@archesproject.org"}
+]
+license = {text = "GNU AGPL3"}
+classifiers = [
+    "Development Status :: 4 - Beta",
```
Comment (aj-he) [inline pyproject.toml:13]: @chiatt - should this be Beta?

Comment (aj-he) [reply pyproject.toml:13]: Would you agree we should be using `5 - Production/Stable`?

https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers

Comment (chiatt) [reply pyproject.toml:13]: We will do that in the stable/1.0.0 branch once this is merged in.

Comment (chiatt) [reply pyproject.toml:13]: We can just leave this in beta so it is ready for patch fixes as soon as stable/1.0.0 is created.

Comment (aj-he) [reply pyproject.toml:13]: Sounds good!

**Review by aj-he — APPROVED** [review]
LGTM

**Pull request merged**